### PR TITLE
Change socket path to fit new path

### PR DESCRIPTION
**Pull request by voladad**
Hyprland devs decided to change the location of the socket file for reasons!

Sorry if this is the wrong way to do this. I have never done any rust programming in my life. This worked for me though so i figured i'd add it here.

### CODE REVIEW

File: src/wm_info_provider/hyprland.rs [hunk 1]
```diff
@@ -1,7 +1,6 @@
 use std::io::{self, Write};
 use std::os::fd::AsRawFd;
 use std::os::unix::net::UnixStream;
-
```
Comment (MaxVerevkin) [inline src/wm_info_provider/hyprland.rs:4]: Unrelated change. Also I like to have a blank line to separate std-imports from the rest.

File: src/wm_info_provider/hyprland.rs [hunk 2]
```diff
@@ -107,8 +106,9 @@ struct Ipc {
 
 impl Ipc {
     fn new(his: &str) -> io::Result<Self> {
-        let sock1_path = format!("/tmp/hypr/{his}/.socket.sock");
-        let sock2_path = format!("/tmp/hypr/{his}/.socket2.sock");
+        let xdgrd = std::env::var("XDG_RUNTIME_DIR").expect("$XDG_RUNTIME_DIR is not set");
```
Comment (MaxVerevkin) [inline src/wm_info_provider/hyprland.rs:109]: I would prefer not to use `.expect()`. Instead `.map_err()`, create an `io::Error` and propagate it. Or, since the error from `Ipc::new()` is ignored anyway, you may change the return type of this function to `Opton<Self>`, in which case you will be able to write `.ok()?` to short-circuit on errors.

**Review by MaxVerevkin — CHANGES_REQUESTED** [review]

**Comment by voladad** [comment]
Yeah i figured i'd mess something up. I'll just close this pull request as its not really needed. Thanks for the feedback

**Comment by MaxVerevkin** [comment]
You didn't mess anything up, I just asked for more graceful error handling. If my comment wasn't clear, you could've asked... 

> as its not really needed

How is it not needed?
